app/javascript/chat/chat.jsx

```diff
@@ -1829,9 +1846,8 @@ export default class Chat extends Component {
     return (
       <div
         data-testid="chat"
-        className={`chat chat--${
-          state.expanded ? 'expanded' : 'contracted'
-        }${detectIOSSafariClass} chat--${
+        className={`chat chat--expanded
```

Comment:
it's possible to even remove the `chat--expanded` class from this entirely.

Reply:
Marie Kondo that if you need to.